Runner/utils/lib_hwprobe.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/sh
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+# hw-probe helpers. Requires lib_common.sh, lib_apt.sh, lib_docker.sh
+
+HWPROBE_PKG="hw-probe"
+HWPROBE_DEPS="lshw smartmontools nvme-cli hdparm pciutils usbutils dmidecode ethtool lsscsi iproute2"
```

Comment:
Looks like hdparm is a Depends of hw-probe already; I wonder how was this list generated? How would it be kept up-to-date?